test/miner_poc_SUITE.erl

```diff
@@ -3,6 +3,7 @@
 -include_lib("common_test/include/ct.hrl").
```

Comment:
sorry to have wasted your time here, but the poc_SUITE has undergone major, incompatible revisions during the pocv4 PR.  can we just revert this whole thing to master for a clean merge both ways?  we can look at refactoring it later, it's done a lot of parallel work to this.

Reply:
will do

Reply:
done